READMEs/c2d-flow.md

```diff
@@ -331,8 +331,9 @@ ALG_order_tx_id = ocean.assets.pay_for_service(
 
 compute_inputs = [ComputeInput(DATA_did, DATA_order_tx_id, compute_service.index)]
 job_id = ocean.compute.start(
-    compute_inputs,
     bob_wallet,
+    compute_inputs[0],
```

Comment:
why are we not sending all the compute inputs? I don't understand.

Reply:
I wouldn't worry about it too much.  I fixed this in https://github.com/oceanprotocol/ocean.py/pull/674

Reply:
I saw that OceanCompute.start() receives `ComputeInput` object, not specifically a list.